src/test/java/com/coherent/task60/TestResultLoggerExtension.java

```diff
@@ -0,0 +1,55 @@
+package com.coherent.task60;
+import org.junit.jupiter.api.extension.AfterAllCallback;
+import org.junit.jupiter.api.extension.ExtensionContext;
+//import org.junit.rules.TestWatcher;
+import org.junit.jupiter.api.extension.TestWatcher;
+import org.junit.runner.Description;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+public class TestResultLoggerExtension implements TestWatcher, AfterAllCallback {
+    private List<TestResultStatus> testResultsStatus = new ArrayList<>();
+    private ScreenshotHelper screenshotHelper = new ScreenshotHelper();
+    private LoginPage loginPage = new LoginPage();
+    private PropertiesHelper propertiesHelper = new PropertiesHelper();
+    private String path = propertiesHelper.propertiesReader("error.screenshots");
```

Comment:
From those variables you need only screenshotHelper

Reply:
Sure. Just forgot to remove them